inc/saiacl.h

```diff
@@ -2148,6 +2151,16 @@ typedef enum _sai_acl_entry_attr_t
      */
     SAI_ACL_ENTRY_ATTR_ACTION_DTEL_REPORT_ALL_PACKETS,
 
+    /**
+     * @brief NAT Pool ID
+     *
+     * @type sai_acl_action_data_t sai_object_id_t
+     * @flags CREATE_AND_SET
+     * @objects SAI_OBJECT_TYPE_NAT
+     * @default disabled
```

Comment:
May be you want to call it NULL and not disabled as its an OID

Reply:
Please review updated diffs